README.md

```diff
@@ -153,16 +153,24 @@ Note: In this case, if the service is staked, then it will not update the on-cha
    cd trader; poetry run python ../report.py; cd ..
    ```
 
-3. Use this command to investigate your agent's logs:
+3. Use the following set of commands to investigate your agent's logs:
 
     ```bash
-    cd trader; poetry run autonomy analyse logs --from-dir trader_service/abci_build/persistent_data/logs/ --agent aea_0 --reset-db; cd ..
+    cd trader
+    service_dir="trader_service"
+    build_dir=$(ls -d "$service_dir"/abci_build_???? 2>/dev/null || echo "$service_dir/abci_build")
+    poetry run autonomy analyse logs --from-dir "$build_dir/persistent_data/logs/" --agent aea_0 --reset-db 
+    cd ..
```

Comment:
I suggest to create a script for this unless @dagacha overrides this decision.

Reply:
It would not make sense to create a script for every possible option. After all, quickstart operators are required to know some basics, such as running a couple of commands.